### PR TITLE
whether can support zookeeper bus and kafka bus #404

### PR DESCRIPTION
**Pull request by bingtianyiyan**
fix： #404 whether can support zookeeper bus and kafka bus

### CODE REVIEW

File: sample/EasyCaching.Demo.Providers/Startup.cs [hunk 1]
```diff
@@ -31,36 +31,68 @@ public void ConfigureServices(IServiceCollection services)
...
+                //});
+
+                //option.UseMemcached(Configuration);
+
+                //kafka bus
+                //use redis cache
                 option.UseRedis(config =>
                 {
-                    config.DBConfig.Endpoints.Add(new ServerEndPoint("127.0.0.1", 6379));
+                    config.DBConfig.Endpoints.Add(new ServerEndPoint("192.168.99.100", 6379));
```
Comment (catcherwong) [inline sample/EasyCaching.Demo.Providers/Startup.cs:69]: maybe should not modify the endpoint of the sample.

File: bus/EasyCaching.Bus.Zookeeper/EasyCaching.Bus.Zookeeper.csproj [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<Import Project="../../build/version.props" />
+	<Import Project="../../build/releasenotes.props" />
+	<PropertyGroup>
+		<TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+		<Owners>ncc;Catcher Wong</Owners>
+		<Authors>ncc;Catcher Wong</Authors>
+		<VersionPrefix>$(EasyCachingZookeeperBusPackageVersion)</VersionPrefix>
```
Comment (catcherwong) [inline bus/EasyCaching.Bus.Zookeeper/EasyCaching.Bus.Zookeeper.csproj:9]: Should add this in `build/version.props`.

File: bus/EasyCaching.Bus.Zookeeper/EasyCaching.Bus.Zookeeper.csproj [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+			A simple caching bus(message bus) based on Kafka.
+		</Description>
+		<PackageTags>Bus,Hybrid,Zookeeper,Caching,Cache</PackageTags>
+		<PackageProjectUrl>https://github.com/dotnetcore/EasyCaching</PackageProjectUrl>
+		<PackageLicenseFile>LICENSE</PackageLicenseFile>
+		<RepositoryUrl>https://github.com/dotnetcore/EasyCaching</RepositoryUrl>
+		<ProjectUrl>https://github.com/dotnetcore/EasyCaching</ProjectUrl>
+		<PackageIcon>nuget-icon.png</PackageIcon>
+		<PackageReleaseNotes>
+			$(EasyCachingKafkaBusPackageNotes)
```
Comment (catcherwong) [inline bus/EasyCaching.Bus.Zookeeper/EasyCaching.Bus.Zookeeper.csproj:21]: Should add this in `build/releasenotes.props`.

File: bus/EasyCaching.Bus.ConfluentKafka/EasyCaching.Bus.ConfluentKafka.csproj [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<Import Project="../../build/version.props" />
+	<Import Project="../../build/releasenotes.props" />
+	<PropertyGroup>
+		<TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+		<Owners>ncc;Catcher Wong</Owners>
+		<Authors>ncc;Catcher Wong</Authors>
+		<VersionPrefix>$(EasyCachingKafkaBusPackageVersion)</VersionPrefix>
```
Comment (catcherwong) [inline bus/EasyCaching.Bus.ConfluentKafka/EasyCaching.Bus.ConfluentKafka.csproj:8]: Should add this in `build/version.props`.

File: bus/EasyCaching.Bus.ConfluentKafka/EasyCaching.Bus.ConfluentKafka.csproj [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+			A simple caching bus(message bus) based on Kafka.
+		</Description>
+		<PackageTags>Bus,Hybrid,Kafka,Caching,Cache</PackageTags>
+		<PackageProjectUrl>https://github.com/dotnetcore/EasyCaching</PackageProjectUrl>
+		<PackageLicenseFile>LICENSE</PackageLicenseFile>
+		<RepositoryUrl>https://github.com/dotnetcore/EasyCaching</RepositoryUrl>
+		<ProjectUrl>https://github.com/dotnetcore/EasyCaching</ProjectUrl>
+		<PackageIcon>nuget-icon.png</PackageIcon>
+		<PackageReleaseNotes>
+			$(EasyCachingKafkaBusPackageNotes)
```
Comment (catcherwong) [inline bus/EasyCaching.Bus.ConfluentKafka/EasyCaching.Bus.ConfluentKafka.csproj:20]: Should add this in `build/releasenotes.props`.

**Review by catcherwong — CHANGES_REQUESTED** [review]

**Comment by catcherwong** [comment]
Please resolve conflicts.

**Comment by bingtianyiyan** [comment]
> Please resolve conflicts.

have been resloved conflicts

**Review by catcherwong — APPROVED** [review]

**Pull request merged**